### ISSUE TITLE
NTR: 4-hydroxybutyric acid high in blood (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8621]
New term request 4-hydroxybutyric acid high in blood (segal_040303194552): gamma-hydroxybutyric acid; GHB

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000991
- label: Elevated circulating 4-hydroxybutyric acid concentration
- parents: [Elevated circulating hydroxybutyric acid concentration (HP:0033417)](https://hpo.jax.org/app/browse/term/HP:0033417)
- pmid: PMID:9683595
- def: Concentration of 4-hydroxybutyric acid in the blood circulation above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8621
- synonyms:
